src/data/injections.js

```diff
@@ -4,7 +4,12 @@
 
 "use strict";
 
-/* globals module */
+/* globals module, require */
+
+// This is a hack for the tests.
+if (typeof getMatchPatternsForGoogleURL === "undefined") {
+  var getMatchPatternsForGoogleURL = require("../lib/google");
```

Comment:
nit: could you please use `const` here as well?

Reply:
No, this is intended to be a global variable declaration. Using `const` instead of `var` would limit the import to the block scope, which will probably cause the test to fail.

This use of `var` (and the comment before it) is copied from src/data/ua_overrides.js .

Reply:
Ahh right, my bad